test/common/scale_up_replicas_rhsso.go

```diff
@@ -67,6 +85,11 @@ func checkScalingOfKeycloakReplicas(t *testing.T, ctx *TestingContext, keycloakC
 	if err := checkNumberOfReplicasAgainstValueRhsso(keycloakCR, ctx, numberOfRhssoReplicas, retryInterval, timeout, t); err != nil {
 		t.Fatalf("Incorrect number of replicas : %v", err)
 	}
+
+	t.Log("Chekcing replicas are ready")
```

Comment:
```suggestion
	t.Log("Checking replicas are ready")
```

Reply:
Thanks for those. will push a new version now.